grades/factories.py

```diff
@@ -0,0 +1,26 @@
+"""Factories for the grades app"""
+from random import randint
+
+import faker
+from factory import SubFactory
+from factory.django import DjangoModelFactory
+
+from courses.factories import CourseRunFactory
+from grades.constants import FinalGradeStatus
+from grades.models import FinalGrade
+from micromasters.factories import UserFactory
+
+
+FAKE = faker.Factory.create()
+
+
+class FinalGradeFactory(DjangoModelFactory):
+    """Factory for FinalGrade"""
+    user = SubFactory(UserFactory)
+    course_run = SubFactory(CourseRunFactory)
+    grade = randint(30, 100)/100.0
+    passed = FAKE.boolean()
```

Comment:
For both `grade` and `passed` here, the values are random initialized staticly, is that the intent?